src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,27 @@
 import React from 'react';
+import { CopyPost } from '../../types/CopyPost';
+import { CommentInfo } from '../CommentInfo';
+import { Comment } from '../../types/Comments';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  post: CopyPost;
+};
+
+export const CommentList: React.FC<Props> = ({ post }) => (
+  <>
+    {post.comments.length > 0 ? (
```

Comment:
you can use deeper destructuring
```suggestion
export const CommentList: React.FC<Props> = ({ post: { comments } }) => (
  <>
    {comments.length > 0 ? (
```